src/relay-utils/src/getDataFromRequest.js

```diff
@@ -1,9 +1,7 @@
 // @flow
 
 import { getRequest, createOperationDescriptor } from 'relay-runtime';
```

Comment:
@tbergq I was thinking. Maybe we should publish another package `@adeira/relay-runtime` solely with the additional Flow types and maybe with some other DX improvements we have? 🤔

Reply:
Not sure if we have some added value over flow-typed when it comes to relay-runtime though.

Reply:
Yeah, if it is just flow-types, we can rather contribute to `flow-typed`

Reply:
I was thinking more and it makes sense to create it. There are other things in runtime like the Store, our Environment factories, networking and similar.